### PR TITLE
E2E test: centralize timeout variable

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Currently E2E are timing out given the time it takes to pull/cache the model may exceed the waiting time

File: hack/e2e_tests.sh [hunk 1]
```diff
@@ -27,6 +27,7 @@ GINKGO_ARGS=${GINKGO_ARGS:-}
 ENABLE_NFD=${ENABLE_NFD:-true}
 ENABLE_GPU_OPERATOR=${ENABLE_GPU_OPERATOR:-true}
 ENABLE_LOCAL_PATH_PROVISIONER=${ENABLE_LOCAL_PATH_PROVISIONER:-true}
+E2E_TIMEOUT=${E2E_TIMEOUT:-"30"} # Int type, will be converted to minutes
```
Comment (shivamerla) [inline hack/e2e_tests.sh:30]: can we rename it to `E2E_TIMEOUT_SECONDS` and default to "1800" to make it explicit.

Comment (ArangoGutierrez) [reply hack/e2e_tests.sh]: Done

File: hack/e2e_tests.sh [hunk 2]
```diff
@@ -27,6 +27,7 @@ GINKGO_ARGS=${GINKGO_ARGS:-}
 ENABLE_NFD=${ENABLE_NFD:-true}
 ENABLE_GPU_OPERATOR=${ENABLE_GPU_OPERATOR:-true}
 ENABLE_LOCAL_PATH_PROVISIONER=${ENABLE_LOCAL_PATH_PROVISIONER:-true}
+E2E_TIMEOUT_SECONDS=${E2E_TIMEOUT_SECONDS:-"1800"} # Int type, will be converted to minutes
```
Comment (shivamerla) [inline hack/e2e_tests.sh:30]: nit: update the comment

Comment (ArangoGutierrez) [reply hack/e2e_tests.sh]: done

**Review by shivamerla — APPROVED** [review]
LGTM

**Pull request merged**